components/server/src/ome/logic/AdminImpl.java

```diff
@@ -641,21 +642,55 @@ public long createSystemUser(Experimenter newSystemUser) {
                 groupProxy(sec.getSecurityRoles().getUserGroupName()));
     }
 
-    @Override
-    @RolesAllowed("system")
-    @Transactional(readOnly = false)
-    public long createLightSystemUser(Experimenter newSystemUser, List<AdminPrivilege> privileges) {
+    // TODO: Consider adding to public API.
```

Comment:
No, don't want to block this PR on it if such a method isn't a blocker for 5.4.0: given recent performance issues, trying to keep my plate otherwise clear, unless @dominikl wants to add.